<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>802</width>
    <height>673</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Lucida Sans</family>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="image.qrc">
    <normaloff>:/icon/picture/file.png</normaloff>:/icon/picture/file.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(210, 228, 238);
color:rgb(65, 101, 121);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPlainTextEdit" name="inputtext">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>761</width>
      <height>541</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cascadia Code</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255, 250, 234);
border-radius: 5px;
border:2px solid rgb(157, 204, 221);
color:black;</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>802</width>
     <height>24</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>Lucida Sans</family>
     <pointsize>10</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QMenu" name="menuopen_file">
    <property name="geometry">
     <rect>
      <x>343</x>
      <y>113</y>
      <width>170</width>
      <height>106</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lucida Sans</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"> QMenu::item:pressed { background-color: #6096B4; color:#EEE9DA }</string>
    </property>
    <property name="title">
     <string>Open File</string>
    </property>
    <addaction name="actionopen_txt_file"/>
    <addaction name="actionopen_cpp_file"/>
   </widget>
   <widget class="QMenu" name="menuanalyse">
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="title">
     <string>Analyse</string>
    </property>
    <widget class="QMenu" name="menuanalyse_txt_file">
     <property name="title">
      <string>analyse txt file</string>
     </property>
     <property name="icon">
      <iconset resource="image.qrc">
       <normaloff>:/icon/picture/statistic.png</normaloff>:/icon/picture/statistic.png</iconset>
     </property>
     <addaction name="actionThe_kth_words"/>
    </widget>
    <widget class="QMenu" name="menuanalyse_cpp_file">
     <property name="title">
      <string>analyse cpp file</string>
     </property>
     <property name="icon">
      <iconset resource="image.qrc">
       <normaloff>:/icon/picture/statistic.png</normaloff>:/icon/picture/statistic.png</iconset>
     </property>
     <addaction name="action_The_frequency_of_the_reserved_words"/>
     <addaction name="actiondetailed_analysis"/>
    </widget>
    <addaction name="menuanalyse_txt_file"/>
    <addaction name="menuanalyse_cpp_file"/>
    <addaction name="separator"/>
    <addaction name="actionsearch"/>
   </widget>
   <widget class="QMenu" name="menusave">
    <property name="font">
     <font>
      <family>Lucida Sans</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"> QMenu::item:pressed { background-color: #6096B4; color:#EEE9DA }</string>
    </property>
    <property name="title">
     <string>Save</string>
    </property>
    <addaction name="actionsave_txt_file"/>
    <addaction name="actionsave_cpp_file"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actioncopy"/>
    <addaction name="actioncut"/>
    <addaction name="actionpaste"/>
    <addaction name="separator"/>
    <addaction name="actionundo"/>
    <addaction name="actionredo"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionabout"/>
   </widget>
   <addaction name="menuopen_file"/>
   <addaction name="menuanalyse"/>
   <addaction name="menuEdit"/>
   <addaction name="menusave"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="font">
    <font>
     <family>Lucida Sans</family>
     <pointsize>10</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(161, 209, 227);
color:rgb(68, 106, 127);
</string>
   </property>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionopen"/>
   <addaction name="actionsave"/>
   <addaction name="separator"/>
   <addaction name="actioncopy"/>
   <addaction name="actioncut"/>
   <addaction name="actionpaste"/>
   <addaction name="actionundo"/>
   <addaction name="actionredo"/>
   <addaction name="actionsearch"/>
   <addaction name="separator"/>
   <addaction name="actionabout"/>
   <addaction name="separator"/>
   <addaction name="actionclose"/>
  </widget>
  <action name="actionopen_txt_file">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/icon/picture/txt.png</normaloff>:/icon/picture/txt.png</iconset>
   </property>
   <property name="text">
    <string>open txt file</string>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
    </font>
   </property>
  </action>
  <action name="actionopen_cpp_file">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/icon/picture/cpp.png</normaloff>:/icon/picture/cpp.png</iconset>
   </property>
   <property name="text">
    <string>open cpp file</string>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
    </font>
   </property>
  </action>
  <action name="actionsearch">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/icon/picture/search.png</normaloff>:/icon/picture/search.png</iconset>
   </property>
   <property name="text">
    <string>search</string>
   </property>
   <property name="toolTip">
    <string>search</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionsave">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/icon/picture/floppy-disk.png</normaloff>:/icon/picture/floppy-disk.png</iconset>
   </property>
   <property name="text">
    <string>save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionsave_cpp_file">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/icon/picture/cpp.png</normaloff>:/icon/picture/cpp.png</iconset>
   </property>
   <property name="text">
    <string>save cpp file</string>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
    </font>
   </property>
  </action>
  <action name="actionsave_txt_file">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/icon/picture/txt.png</normaloff>:/icon/picture/txt.png</iconset>
   </property>
   <property name="text">
    <string>save txt file</string>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
    </font>
   </property>
  </action>
  <action name="actionopen">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/icon/picture/open-folder.png</normaloff>:/icon/picture/open-folder.png</iconset>
   </property>
   <property name="text">
    <string>actionopen</string>
   </property>
   <property name="toolTip">
    <string>open file</string>
   </property>
  </action>
  <action name="actionThe_kth_words">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/icon/picture/list.png</normaloff>:/icon/picture/list.png</iconset>
   </property>
   <property name="text">
    <string>The kth words</string>
   </property>
  </action>
  <action name="action_The_frequency_of_the_reserved_words">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/icon/picture/list.png</normaloff>:/icon/picture/list.png</iconset>
   </property>
   <property name="text">
    <string>frequency of reserved words</string>
   </property>
  </action>
  <action name="actiondetailed_analysis">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/icon/picture/side-menu.png</normaloff>:/icon/picture/side-menu.png</iconset>
   </property>
   <property name="text">
    <string>detailed analysis</string>
   </property>
  </action>
  <action name="actionredo">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/icon/picture/redo.png</normaloff>:/icon/picture/redo.png</iconset>
   </property>
   <property name="text">
    <string>redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="actionundo">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/icon/picture/undo.png</normaloff>:/icon/picture/undo.png</iconset>
   </property>
   <property name="text">
    <string>undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionclose">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/icon/picture/logout.png</normaloff>:/icon/picture/logout.png</iconset>
   </property>
   <property name="text">
    <string>close</string>
   </property>
  </action>
  <action name="actionpaste">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/icon/picture/paste.png</normaloff>:/icon/picture/paste.png</iconset>
   </property>
   <property name="text">
    <string>paste</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actioncut">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/icon/picture/scissors.png</normaloff>:/icon/picture/scissors.png</iconset>
   </property>
   <property name="text">
    <string>cut</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actioncopy">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/icon/picture/copy.png</normaloff>:/icon/picture/copy.png</iconset>
   </property>
   <property name="text">
    <string>copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionabout">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/icon/picture/information.png</normaloff>:/icon/picture/information.png</iconset>
   </property>
   <property name="text">
    <string>about</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="image.qrc"/>
 </resources>
 <connections/>
</ui>
